bokeh/embed.py

```diff
@@ -40,35 +43,91 @@ def components(plot_object, resources=None):
               are **already loaded**.
 
     Args:
-        plot_object (PlotObject) : Bokeh object to render
-            typically a Plot or PlotContext
+        plot_objects (PlotObject|list|dict|tuple) :
+        The |components| function takes either a single PlotObject, a list/tuple of
+        PlotObjects, or a dictionary of keys and PlotObjects. Each returns
+        a corresponding data structure of script and div pairs.
+
+        The following illustrates how different input types correlate to outputs:
+
+            components(plot)
+            #=> (script, plot_div)
+
+            components((plot_1, plot_2))
+            #=> (script, (plot_1_div, plot_2_div))
+
+            components({"Plot 1": plot_1, "Plot 2": plot_2})
+            #=> (script, {"Plot 1": plot_1_div, "Plot 2": plot_2_div})
+
+        An example can be found in examples/embed/embed_multiple.py
+
         resources : Deprecated argument
     Returns:
-        (script, div) : UTF-8 encoded
-
+        (script, div[s]): UTF-8 encoded
     '''
-    
+    from .document import Document
```

Comment:
This is local because of a circular dependency problem I can't nail down. Any thoughts would be appreciated.